specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -44,6 +44,16 @@ openapi-type: arm
 tag: package-2019-06-preview
```

Comment:
Will you use new api version for your SDK release? if yes, we should change tag to your new tag for 2021-01 version

Reply:
Hi,
no . We don't want to use the new API version for the SDK release